docs/rbac.rst

```diff
@@ -0,0 +1,260 @@
+         "rpm.view_rpmalternatecontentsource",
+         "rpm.view_rpmdistribution",
+         "rpm.view_rpmpublication",
+         "rpm.view_rpmremote",
+         "rpm.view_rpmrepository",
+         "rpm.view_ulnremote",
+      ],
+      "locked": true
+    }
+
+.. important::
+
+   Please note there are two types of Remotes with their permissions.
+
+
+RPM Viewer Role (rpm.viewer)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A universal role allows you to view all content within the RPM plugin.
+
+.. code-block:: shell
+   :caption: Universal RPM viewer role
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/d1e7bae0-2363-45e2-815c-d721ecc3133c/",
+      "pulp_created": "2022-04-07T12:41:11.144883Z",
+      "name": "rpm.viewer",
+      "description": null,
+      "permissions": [
+        "rpm.view_rpmalternatecontentsource",
+        "rpm.view_rpmdistribution",
+        "rpm.view_rpmpublication",
+        "rpm.view_rpmremote",
+        "rpm.view_rpmrepository",
+        "rpm.view_ulnremote",
+      ],
+      "locked": true
+    }
+
+
+Creating Roles
+--------------
+
+.. note::
+
+    Roles shipped by the RPM plugin mentioned above are locked and cannot be modified or removed.
+
+To create a new role, you need to specify its name and permissions to use.
+To find out which permissions you need for a new role, you can list an endpoints policies with the following command:
+
+.. code-block:: shell
+   :caption: List an access policy of the repository endpoint
+
+    pulp access-policy show --viewset-name "repositories/rpm/rpm"
+
+**Example 1: RPM Repository Creator**
+
+.. code-block:: shell
+   :caption: Example of the RPM repository creator role
+
+    pulp role create --name "rpm_repo_creator" \
+        --permission "rpm.add_rpmrepository" \
+        --permission "rpm.view_rpmrepository" \
+        --permission "rpm.sync_rpmrepository" \
+        --permission "rpm.modify_rpmrepository" \
+        --permission "rpm.change_rpmrepository" \
+        --permission "rpm.delete_rpmrepository" \
+        --permission "rpm.add_rpmremote" \
+        --permission "rpm.change_rpmremote" \
+        --permission "rpm.delete_rpmremote" \
+        --permission "rpm.view_rpmremote"
+
+**Example 2: RPM Publisher**
+
+.. code-block:: shell
+   :caption: Example of the RPM publisher role
+
+    pulp role create --name "rpm_publisher" \
+        --permission "rpm.view_rpmrepository" \
+        --permission "rpm.add_rpmpublication" \
+        --permission "rpm.add_rpmdistribution"
+
+.. code-block:: shell
+
+    pulp user role-assignment add --username "bob" --role "rpm_repo_creator" --object ""
+    pulp user role-assignment add --username "jack" --role "rpm_publisher" --object ""
+
+.. warning::
+
+   If a role contains multiple permissions over different objects, then you can't assign this role to a specific object.
+   Trying to assign the role to a single object will fail.
+   For example, if a role contains ``add`` permission, you must use the empty ``--object`` param.
```

Comment:
This is not true.
Having the `add` permission on an object does not lead anywhere, but it's possible.
Also assigning a role to an object is possible if at least one of it's permissions is applicable to that object.